framework/src/base/FEProblem.C

```diff
@@ -3691,6 +3691,9 @@ FEProblem::adaptMesh()
       computeMarkers();
     if (_adaptivity.adaptMesh())
       meshChanged();
+
+    // Show adaptivity progress
+    _console << std::flush;
```

Comment:
@jessecarterMOOSE - This is tricky to test. Does it appear to be working for you?